### PR TITLE
layout and navbar

### PR DESCRIPTION
**Pull request by omerzamir**
What do you say about it?
@omrishtam @shaySirek @RanNissan 

Signed-off-by: Omer Zamir <zamir98@gmail.com>

### CODE REVIEW

**Comment by omerzamir** [comment]
I made a template for dependents like user that i dont have right now...

File: Views/Shared/_Weather.cshtml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                },
+
+                function fail(data, status) {
+                    console.error('Request failed.  Returned status ', status);
+                    reject(status);
+                }
+            );
+    }
+
+    function getPosition() {
```
Comment (omerzamir) [inline Views/Shared/_Weather.cshtml:38]: Function Explanation:
Trying to get location from browser api, if the browser isn't supporting this feature or user hasn't have is access to his location, we are trying to get his location by ip(less accurate).

**Comment by RanNissan** [comment]
Add screenshots of your project

**Comment by shaySirek** [comment]
@omerzamir  In my opinion, the layout looks pretty good.

**Pull request merged**